src/dialogs/confirmCallbackStep.ts

```diff
@@ -45,10 +44,12 @@ export class ConfirmCallbackStep extends ComponentDialog {
     let okMsg;
     let standardMsg;
 
+    console.log(stepContext)
+
     switch(stepContext.stack[0].id){
       case 'CONFIRM_CALLBACK_WATERFALL_STEP':
         okMsg = false;
-        standardMsg = i18n.__('confirmHomeAddressFailedErrorMsg');
+        standardMsg = i18n.__('unblock_direct_deposit_main_error');
```

Comment:
not sure why here change the standard message

Reply:
this is a portion I need help with, i'll explain in a different PR. This is a TODO.